wdl/model/draft2/src/main/scala/wdl/draft2/model/Scope.scala

```diff
@@ -41,17 +43,20 @@ trait Scope {
   }
 
   lazy val childGraphNodes: Set[WdlGraphNode] = children.toSet.filterByType[WdlGraphNode]
-  lazy val childGraphNodesSorted: List[WdlGraphNode] = {
+  lazy val childGraphNodesSorted: ErrorOr[List[WdlGraphNode]] = {
```

Comment:
IMO this should be `Either` as `Validated` is not a Monad and we are flatMap-ing in violation of FP statute 1,000,004.

(Acknowledging this has ripple effects upward and would be painful)